src/lavinmq/config.cr

```diff
@@ -62,6 +62,7 @@ module LavinMQ
     property default_consumer_prefetch = UInt16::MAX
     property yield_each_received_bytes = 131_072    # max number of bytes to read from a client connection without letting other tasks in the server do any work
     property yield_each_delivered_bytes = 1_048_576 # max number of bytes sent to a client without tending to other tasks in the server
+    property auth_backends : Array(String)? = ["basic"]
```

Comment:
```suggestion
    property auth_backends : Array(String) = ["basic"]
```
No need to be nilable when it has a default value

Reply:
That default was mostly put there for manual testing hehe, idk if we should have a default value like this? if we do, we don't need to have the safety clause in the chain that checks if its empty etc